src/main/java/frc/robot/commands/IntakeToOuttake.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.IntakeSubsystem;
+public class IntakeToOuttake extends Command {
+  IntakeSubsystem m_intakeSubsystem;
+  /** Creates a new IntakesOutake. */
+  public IntakeToOuttake(IntakeSubsystem subsystem) {
+    m_intakeSubsystem = subsystem;
+    addRequirements(subsystem);
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+        m_intakeSubsystem.spit();
+
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {}
```

Comment:
You forgot to stop moving the intake when it ends

Reply:
fixed